__tests__/app.test.js

```diff
@@ -40,8 +40,57 @@ describe("GET /api/categories", () => {
     return request(app)
       .get("/api/bad_path")
       .expect(404)
-      .then(({body: {msg} }) => {
+      .then(({ body: { msg } }) => {
         expect(msg).toBe("Invalid Path");
       });
   });
 });
+
+describe("GET /api/reviews", () => {
+  describe.only("GET /api/reviews/:review_id", () => {
+    test("status 200: responds with review object with relevant keys ", () => {
+      return request(app)
+        .get("/api/reviews/3")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body).toEqual({
+            review_id: 3,
+            title: "Ultimate Werewolf",
+            category: "social deduction",
+            designer: "Akihisa Okui",
+            owner: "bainesface",
+            review_body: "We couldn't find the werewolf!",
+            review_img_url:
+              "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+            created_at: "2021-01-18T10:01:41.251Z",
+            votes: 5,
+          });
+          expect(body).toHaveProperty("review_id");
+          expect(body).toHaveProperty("title");
+          expect(body).toHaveProperty("review_body");
+          expect(body).toHaveProperty("designer");
+          expect(body).toHaveProperty("review_img_url");
+          expect(body).toHaveProperty("votes");
+          expect(body).toHaveProperty("category");
+          expect(body).toHaveProperty("owner");
+          expect(body).toHaveProperty("created_at");
```

Comment:
This is unncessary code as you are testing for all of these things in a more specific way above by checking it equals the exact values of the above object